config/locales/doc_auth/en.yml

```diff
@@ -179,6 +174,11 @@ en:
         address.
       tag: Recommended
       upload: We’ll collect information about you by reading your state-issued ID.
+      upload_from_computer: Don’t have a phone? Upload photos of your ID from this computer.
+      upload_from_phone: You won’t have to sign in again, and you’ll switch back to
+        this computer after you take photos. Your mobile phone must have a
+        camera and a web browser.
+
```

Comment:
I'm surprised the linter didn't take issue with the extra newline here. Maybe it's treated as significant whitespace? In any case, it didn't exist previously and there's not a convention for it, so I expect it should be removed.

```suggestion
```